pkg/middleware/jwt_session.go

```diff
@@ -60,7 +62,11 @@ func (j *jwtSessionLoader) loadSession(next http.Handler) http.Handler {
 // getJwtSession loads a session based on a JWT token in the authorization header.
 // (see the config options skip-jwt-bearer-tokens and extra-jwt-issuers)
 func (j *jwtSessionLoader) getJwtSession(req *http.Request) (*sessionsapi.SessionState, error) {
-	auth := req.Header.Get("Authorization")
+	authHeader := "Authorization"
+	if j.jwtAuthHeader == "" {
```

Comment:
Do we ever use the header from the config in this code (:warning: I have no Go knowledge whatsoever…)

I would assume something like this instead.
```go
	authHeader := "Authorization"
	if j.jwtAuthHeader != nil && len(strings.TrimSpace(j.jwtAuthHeader)) > 0 {
		authHeader = j.jwtAuthHeader
	}
```